vmdb/lib/evm_database.rb

```diff
@@ -17,9 +17,9 @@ class EvmDatabase
     VmdbDatabase
   }
 
-  def self.model_class_names
-    @model_class_names ||= begin
-      Dir.glob(File.join(Rails.root, "app/models/*.rb")).collect { |f| File.basename(f, ".*").camelize }.compact.sort
+  def self.seedable_model_class_names
+    @seedable_model_class_names ||= begin
+      Dir.glob(Rails.root.join("app/models/*.rb")).collect { |f| File.basename(f, ".*").camelize if File.read(f).include?("self.seed") }.compact.sort
```

Comment:
@tenderlove @matthewd @jrafanie Wanted to point you to this, which is really interesting.  In order to seed, we load all of the models, so even though we lazy load everything, we end up loading everything anyway!  Maybe the whole lazy loading thing needs reevaluation especially in light of this.